client/state/action-types.js

```diff
@@ -194,7 +194,12 @@ export const KEYRING_SERVICES_REQUEST_SUCCESS = 'KEYRING_SERVICES_REQUEST_SUCCES
 export const LAYOUT_FOCUS_SET = 'LAYOUT_FOCUS_SET';
 export const LAYOUT_NEXT_FOCUS_SET = 'LAYOUT_NEXT_FOCUS_SET';
 export const LAYOUT_NEXT_FOCUS_ACTIVATE = 'LAYOUT_NEXT_FOCUS_ACTIVATE';
+export const MEDIA_ITEMS_RECEIVE = 'MEDIA_ITEMS_RECEIVE';
```

Comment:
>To support that, in addition to `MEDIA_ITEMS_RECEIVE` we'll probably have to dispatch a `MEDIA_ITEM_RECEIVE` for each single media item.

It's not clear from your comment, but what would a separate `MEDIA_ITEM_RECEIVE` action enable that couldn't be achieved by looping over items of `MEDIA_ITEMS_RECEIVE` ?

Ideally trying to avoid multiple actions to simplify tests and implementation of reducers. See sites [as an example](https://github.com/Automattic/wp-calypso/blob/207d4eda3c6a8b999cb270f5c63348cacbd34f64/client/state/current-user/reducer.js#L63-L84) of why this can be a pain.